### PR TITLE
latest cut

### PR DESCRIPTION
**Pull request by iancrowther**

### CODE REVIEW

File: slides/january-2016/deck.md [hunk 1]
```diff
@@ -160,11 +198,12 @@ master: bullet-caption-slide
 -->
 
 # Next Time
-## January 27th 2016
+## February 24th 2016
 The 4th Wednesday of the month
 
 * Rubbish in Bins
 * Arrange Tables & Chairs
+* Visit Sponsors Websites
```
Comment (clarkie) [inline slides/january-2016/deck.md]: What's this? A call to action for attendees to find out who our sponsors are?

File: slides/january-2016/deck.md [hunk 2]
```diff
@@ -129,7 +167,7 @@ master: image
 # Loading...
 Grab a drink whilst we setup!
 
-![](images/break.gif)
+![](images/...)
```
Comment (clarkie) [inline slides/january-2016/deck.md]: these aren't going to work

Comment (clarkie) [inline slides/january-2016/deck.md]: ignore that it's pending!

File: slides/january-2016/deck.md (hunk 1 above)
Comment (iancrowther) [inline slides/january-2016/deck.md]: just a reminder to visit websites.. as a thanks to the sponsors

File: slides/january-2016/deck.md (hunk 2 above)
Comment (iancrowther) [inline slides/january-2016/deck.md]: yep - ...'s need finding

**Comment by clarkie** [comment]
+1 looks good

**Pull request merged**